libr/core/cmd_seek.c

```diff
@@ -273,30 +273,41 @@ static void seek_to_register(RCore *core, const char *input, bool is_silent) {
 }
 
 static int cmd_seek_opcode_backward(RCore *core, int n) {
-	int val = 0;
+	int i, val = 0;
 	// N previous instructions
 	ut64 addr = core->offset;
 	int ret = 0;
 	int numinstr = n * -1;
 	if (r_core_prevop_addr (core, core->offset, numinstr, &addr)) {
 		ret = core->offset - addr;
+		r_core_seek (core, addr, true);
```

Comment:
I think this should stay where it was, after the `if/else` because it's done in both branches.

Reply:
yes i think the same, but if i do that, then it seeks to UT64_MAX, even with asan or valgrind. thats the heissenbug ive been trying to fix until i gave up

Reply:
try it yourself if you have time otherwise ill have a look later today when i get some more time for this

Reply:
that's interesting. it should be exactly the same from the execution point of view (unless some race condition with the `task` stuff)